packages/app-store/zapier/pages/setup/index.tsx

```diff
@@ -18,7 +18,7 @@ export default function ZapierSetup(props: IZapierSetupProps) {
   const [newApiKeys, setNewApiKeys] = useState<Record<string, string>>({});
   const { t } = useLocale();
   const utils = trpc.useContext();
-  const integrations = trpc.viewer.integrations.useQuery({ variant: "automation" });
+  const integrations = trpc.viewer.integrations.useQuery({ categories: ["automation"] });
```

Comment:
@joeauyeung Have you carefully checked that in every config.json or meta.ts, categories had the variant set in it.